modules/deploy/pages/deployment-option/self-hosted/manual/production/production-readiness.adoc

```diff
@@ -0,0 +1,558 @@
+= Production Readiness Checklist 
+
+Before running production workloads on Redpanda, work through the readiness checklists provided here to ensure that your cluster is deployed and configured according to Redpanda Data recommendations. The checklists help ensure that you are set up for success. There are three levels of production readiness:
+- Level 1 
+** Description of the level.
+- Level 2
+** Description of the level.
+- Level 3 
+** Description of the level.
```

Comment:
```suggestion
- * Level 1 
Level 1 readiness checks that all necessary defaults and configuration items have been set, that the hardware has been setup optimally, security is enabled and that you are set up for running in production.
- * Level 2
Level 2 checks that best practices are being followed in terms of day 2 operations, and that you have sufficient ability to diagnose and recover from failures or problems.
- * Level 3 
Level 3 is full enterprise readiness for running mission-critical workloads. This comprises, proactive monitoring and alerting, business continuity solutions and integration into enterprise security systems. 
```

Reply:
@Deflaimun Please move these descriptions under the heading for each checklist. They can then serve as an introduction to the checklist and help users understand how we are defining the levels.  
Also, format as:
For Level 1 readiness, check:
- That all required defaults and configuration items are specified
- You have the optimal setup of the hardware
- Security is enabled
- You are set up to run in production